### PR TITLE
add web post redirect uri config

### PR DESCRIPTION
**Pull request by micbar**
## Description

Enhancement: Make the post logout redirect uri configurable

We added a config option to change the redirect uri after the logout action of the web client.

https://github.com/owncloud/ocis/issues/6536

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: services/web/pkg/config/config.go [hunk 1]
```diff
@@ -65,11 +65,12 @@ type WebConfig struct {
...
-	Authority    string `json:"authority,omitempty" yaml:"authority" env:"OCIS_URL;OCIS_OIDC_ISSUER;WEB_OIDC_AUTHORITY" desc:"URL of the OIDC issuer. It defaults to URL of the builtin IDP."`
-	ClientID     string `json:"client_id,omitempty" yaml:"client_id" env:"OCIS_OIDC_CLIENT_ID;WEB_OIDC_CLIENT_ID" desc:"OIDC client ID, which ownCloud Web uses. This client needs to be set up in your IDP (has no effect when using the builtin IDP)."`
-	ResponseType string `json:"response_type,omitempty" yaml:"response_type" env:"WEB_OIDC_RESPONSE_TYPE" desc:"OIDC response type to use for authentication."`
-	Scope        string `json:"scope,omitempty" yaml:"scope" env:"WEB_OIDC_SCOPE" desc:"OIDC scopes to request during authentication to authorize access to user details. Defaults to 'openid profile email'. Values are separated by blank. More example values but not limited to are 'address' or 'phone' etc."`
+	MetadataURL           string `json:"metadata_url,omitempty" yaml:"metadata_url" env:"WEB_OIDC_METADATA_URL" desc:"URL for the OIDC well-known configuration endpoint. Defaults to the oCIS API URL + \"/.well-known/openid-configuration\"."`
```
Comment (mmattel) [inline services/web/pkg/config/config.go:68]:
```suggestion
	MetadataURL           string `json:"metadata_url,omitempty" yaml:"metadata_url" env:"WEB_OIDC_METADATA_URL" desc:"URL for the OIDC well-known configuration endpoint. Defaults to the oCIS API URL + '/.well-known/openid-configuration'."`
```
Single quotes increase the readbility

File: services/web/pkg/config/config.go [hunk 2]
```diff
@@ -65,11 +65,12 @@ type WebConfig struct {
...
+	Authority             string `json:"authority,omitempty" yaml:"authority" env:"OCIS_URL;OCIS_OIDC_ISSUER;WEB_OIDC_AUTHORITY" desc:"URL of the OIDC issuer. It defaults to URL of the builtin IDP."`
+	ClientID              string `json:"client_id,omitempty" yaml:"client_id" env:"OCIS_OIDC_CLIENT_ID;WEB_OIDC_CLIENT_ID" desc:"OIDC client ID, which ownCloud Web uses. This client needs to be set up in your IDP (has no effect when using the builtin IDP)."`
```
Comment (mmattel) [inline services/web/pkg/config/config.go:70]:
```suggestion
	ClientID              string `json:"client_id,omitempty" yaml:"client_id" env:"OCIS_OIDC_CLIENT_ID;WEB_OIDC_CLIENT_ID" desc:"The OIDC client ID which ownCloud Web uses. This client needs to be set up in your IDP. Note that this setting has no effect when using the builtin IDP."`
```

File: services/web/pkg/config/config.go [hunk 3]
```diff
@@ -65,11 +65,12 @@ type WebConfig struct {
...
+	ResponseType          string `json:"response_type,omitempty" yaml:"response_type" env:"WEB_OIDC_RESPONSE_TYPE" desc:"OIDC response type to use for authentication."`
```
Comment (mmattel) [inline services/web/pkg/config/config.go:71]:
```suggestion
	ResponseType          string `json:"response_type,omitempty" yaml:"response_type" env:"WEB_OIDC_RESPONSE_TYPE" desc:"The OIDC response type to use for authentication."`
```

**Comment by micbar** [comment]
@mmattel you are scope creeping my PR, you suggestions are applying to stuff which is not part of this PR

**Comment by mmattel** [comment]
The files view show that this got changed. I take care that the docs part for the changes shown is correct.

**Comment by micbar** [comment]
I only added one line, the diff is bigger because the indentation has changed.

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>Core-API-Tests-ocis-storage-3</strong>](https://drone.owncloud.com/owncloud/ocis/23543/60/1) failed. Further test are cancelled...

**Review by mmattel — APPROVED** [review]
LGTM from a docs pov 👍

**Comment by micbar** [comment]
@JammingBen Can we get an approval from the web team?

**Review by JammingBen — APPROVED** [review]
👍

**Pull request merged**